.github/pull_request_template.md

```diff
@@ -2,7 +2,7 @@ Please answer these questions before submitting your pull requests. Thanks!
 
 1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.
 
-   Fixes #NNNN
+   Fixes SNOW-NNNNNNN
```

Comment:
This this is a public repository, I think we want it to be generic so github issues can be referenced.